aspnetcore/blazor/forms-and-input-components.md

```diff
@@ -37,28 +37,90 @@ The <xref:Microsoft.AspNetCore.Components.Forms?displayProperty=fullName> namesp
 
 A project created from the Blazor project template includes the namespace by default in the app's `_Imports.razor` file, which makes the namespace available to the app's Razor components.
 
+:::moniker range=">= aspnetcore-8.0"
+
+## Enhanced navigation and form handling
```

Comment:
This article is about forms, not navigation, so I think we can remove navigation from this heading.

```suggestion
## Enhanced form handling
```

I also think this section should go much later in the article, probably near the end. Enhanced form handling is totally optional and we don't want to scare off new users with these details right from the start. We can teach them all they need to know about form without any mention of enhanced form handling and then at the end let them know that they can ✨enhance✨ their Blazor forms by adding the appropriate attribute and explain what that does and how it interacts with redirects to non-Blazor pages.

Reply:
I think we can replace "enhanced navigation from a form" with "enhanced form handling" in this section.